scripts/release.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
CONTRIBUTING.md points at `make release`, which I think should now run this script instead of goreleaser

Reply:
oops! i meant to change `make release` to point to this script. updated the release documentation!